config/routes.rb

```diff
@@ -4,6 +4,9 @@
   mount Rswag::Api::Engine => "/"
   mount Rswag::Ui::Engine  => "api-docs"
 
+  get "link_exchange/*path", to: redirect { |params, request| "https://tulle.tul-infra.page/#{params[:path]}" }
```

Comment:
@dkinzer Can you also add the following?

`get "r/*path", to: redirect { |params, request| "https://tulle.tul-infra.page/r/#{params[:path]}" }`

These are redirect links that are generated by the Link Exchanger Service.

Reply:
Actually, forget that. Let's first see if it works without this change.